src/modules/mints/MintVolumeChart.tsx

```diff
@@ -0,0 +1,36 @@
+import moment from "moment";
+import { FunctionComponent } from "react";
+import { MintEntities } from "../../types/entities";
+import { BarChart } from "../../uikit/charts/BarChart";
+
+type MintVolumeChartProps = {
+	mintEntities: MintEntities[];
+};
+
+const MintVolumeChart: FunctionComponent<MintVolumeChartProps> = ({
+	mintEntities,
+}) => {
+	const mintAmountGroupByDate: { [day: string]: number } = {};
+	mintEntities.forEach((item) => {
+		const day: string = moment
+			.unix(parseInt(item.timestamp))
+			.format("YYYY-MM-DD");
+		const addedAmount: number = parseFloat(item.amountRISE) / 10 ** 18;
+		if (!mintAmountGroupByDate[day]) mintAmountGroupByDate[day] = 0;
+		mintAmountGroupByDate[day] += addedAmount;
+	});
+	const mintedVolume = Object.values(mintAmountGroupByDate).reverse();
+	const minterTimestamps = Object.keys(mintAmountGroupByDate)
+		.map((item) => moment(item).format("MMMM D, YYYY"))
```

Comment:
it would be better if you create helper function 

so when you need this kind of format, you not writing it twice.